src/components/LandingVideo/LandingVideo.js

```diff
@@ -0,0 +1,34 @@
+import "./LandingVideo.scss";
+
+import appstore from "../../assets/appstore.png";
+import iphone6 from "../../assets/iphone6.png";
+import facebook from "../../assets/icons/facebook.svg";
+import twitter from "../../assets/icons/twitter.svg";
+
+const LandingVideo = () => {
+  return (
```

Comment:
```suggestion
const LandingVideo = () => (
```
In these cases you can implicitly return